service/src/main/kotlin/no/nav/su/se/bakover/service/statistikk/mappers/BehandlingStatistikkMapper.kt

```diff
@@ -162,16 +162,94 @@ class BehandlingStatistikkMapper(
                 }
                 is UnderkjentRevurdering -> {
                     copy(
-                        beslutter = revurdering.attestering.attestant.navIdent
+                        beslutter = revurdering.attestering.attestant.navIdent,
                     )
                 }
                 else -> throw ManglendeStatistikkMappingException(this, revurdering::class.java)
             }
         }
     }
 
+    fun map(gjenopptak: GjenopptaYtelseRevurdering): Statistikk.Behandling {
+        Statistikk.Behandling(
+            behandlingType = Statistikk.Behandling.BehandlingType.REVURDERING,
+            behandlingTypeBeskrivelse = Statistikk.Behandling.BehandlingType.REVURDERING.beskrivelse,
+            funksjonellTid = Tidspunkt.now(clock),
```

Comment:
Hvorfor gjør vi ikke noe toOzloTime for funksjonellTid og tekniskTid? Antar det er et generelt spm.

Reply:
Godt spørsmål, kunde ha varit det men blev nog aldrig gjort sån bara.